app/src/main/java/org/kiwix/kiwixmobile/main/KiwixWebViewClient.java

```diff
@@ -108,9 +114,30 @@ private void inflateHomeView(WebView view) {
     LayoutInflater inflater = LayoutInflater.from(view.getContext());
     home = inflater.inflate(R.layout.content_main, view, false);
     callback.setHomePage(home);
+
     if (sharedPreferenceUtil.nightMode()) {
+
       ImageView cardImage = home.findViewById(R.id.content_main_card_image);
-      cardImage.setImageResource(R.drawable.ic_home_kiwix_banner_night);
+      ImageView sideImage = home.findViewById(R.id.content_side_image);
+      AppCompatButton downloadButton = home.findViewById(R.id.content_main_card_download_button);
+
+      downloadButton.setTextColor(Color.parseColor("#000000"));
+      if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+        downloadButton.setBackgroundTintList(ColorStateList.valueOf(
+            KiwixApplication.getInstance().getResources().getColor(R.color.complement_blue800)));
+      }
+
+      ConstraintLayout constraintLayout = home.findViewById(R.id.constraint_main);
+      constraintLayout.setBackgroundResource(R.drawable.back_cover_night);
+      sideImage.setImageResource(R.drawable.home_side_cover_night);
+
+      if (KiwixApplication.getInstance().getResources().getConfiguration().orientation ==
```

Comment:
Earlier you were using layout resource like this. You can also do the same with `drawables` or any other resources. You can read this https://developer.android.com/guide/topics/resources/providing-resources for more info.